src/js/HomeLink.jsx

```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+var Link = require('react-router').Link;
+
+var HomeLink = React.createClass({
+  render: function(){
+    return <Link className="HomeLink" to="/">Home</Link>
```

Comment:
I kinda like having the components name as the classname on the node it returns. Seems convenient for debugging and such, and any specific styling outside the general grid system, etc. Doesn't really cost us anything.